README.md

```diff
@@ -121,6 +121,7 @@ Templating variables:
 %c       Print story creation timestamp
 %updated Print story updated timestamp (if different from created)
 %j       Print full story as formatted JSON
+%git     Print Git Integration branch
```

Comment:
I'm open to other formatting key naming here

Reply:
```suggestion
%gb     Print Git Integration branch
```

How about `gb` for branch?

Reply:
Love it. Makes more sense given what we're returning.